libwallet/src/internal/selection.rs

```diff
@@ -55,6 +55,10 @@ where
 	C: NodeClient + 'a,
 	K: Keychain + 'a,
 {
+	//TODO: Revise HF3. If we're sending V4 slates, only include
+	// change outputs in excess sum
+	let include_inputs_in_sum = if slate.is_compact() { false } else { true };
```

Comment:
This is an eye sore to me:-(
 Please change to

let include_inputs_in_sum = !slate.is_compact();

Reply:
Truly a classic. No idea where that evolved from.

Reply:
Surely this should be -

```
if !slate.is_compact() { true } else { false };
```

Reply:
No, inputs aren't included in the participant's excess calculation at this stage if the slate is v4 (read `is_compact` == `is_v4). The input excesses are subtracted from the offset later in the process.